src/region.js

```diff
@@ -232,7 +184,7 @@ var Region = MarionetteObject.extend({
     // present before replacing
     this._restoreEl();
 
-    var parent = this.el.parentNode;
+    const parent = this.el.parentNode;
 
     parent.replaceChild(view.el, this.el);
     this.replaced = true;
```

Comment:
Done.  I exposed with `isReplaced()` in keeping with view state methods, but I'm not sure that's essential public information.